.github/workflows/autoupdate.yml

```diff
@@ -0,0 +1,51 @@
+name: Update
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  update:
+    strategy:
+      matrix:
+        project:
+          - 'scipp/esssans'
+          - 'scipp/sciline'
+          - 'scipp/scippnexus'
+          - 'scipp/scippneutron'
+          - 'scipp/plopp'
+          - 'scipp/essnmx'
+          - 'scipp/essreduce'
+          - 'scipp/beamlime'
+          - 'scipp/essspectroscopy'
+          - 'scipp/chexus'
+          - 'scipp/ess'
+          - 'scipp/chexus'
+          - 'scipp/essreflectometry'
+          - 'scipp/essdiffraction'
+          - 'scipp/esspolarization'
+          - 'scipp/scippuncertainty'
+          - 'scipp/essimaging'
```

Comment:
Please do comment if something is missing here.

Reply:
[cyclebane](https://github.com/scipp/cyclebane)?

Reply:
And as I mentioned above, can we set them from an input parameter of the job?
I'm not sure if github allows iterable type of input though... Maybe not... 

Then we might have to make a re-usable atomic action first and then use matrix on them, like the [ci.yml](https://github.com/scipp/copier_template/blob/main/template/.github/workflows/ci.yml) uses [test.yml](https://github.com/scipp/copier_template/blob/main/template/.github/workflows/test.yml)